set.mm

```diff
@@ -91280,6 +91280,47 @@ without using the Axiom of Replacement or the Axiom of Power Sets (unlike
     UHABCDOUDUJLCUGJKZUHCUGUIPUDUKUHSUJCUGQRTUAUBUC $.
   $( $j usage 'f1imaenfi' avoids 'ax-pow' 'ax-rep'; $)
 
+  ${
+    $d D x $.  $d H x $.  $d A f g x $.  $d B f g x $.
+    sbthfilem.1 $e |- A e. _V $.
+    sbthfilem.2 $e |- D = { x |
+                   ( x C_ A /\ ( g " ( B \ ( f " x ) ) ) C_ ( A \ x ) ) } $.
+    sbthfilem.3 $e |- H = ( ( f |` U. D ) u. ( `' g |` ( A \ U. D ) ) ) $.
+    sbthfilem.4 $e |- B e. _V $.
+    $( Lemma for ~ sbthfi .  (Contributed by BTernaryTau, 4-Nov-2024.) $)
+    sbthfilem $p |- ( ( B e. Fin /\ A ~<_ B /\ B ~<_ A ) -> A ~~ B ) $=
+      ( cfn wcel cdom wbr w3a cv wex wa cvv wf1 cen 19.42vv 3anass 2exbii brdom
+      anbi12i exdistrv bitr4i anbi2i 3bitr4ri wf1o wfn f1fn cuni cres ccnv cdif
+      bitri cun vex resex fnfi cnvfi resexg 3syl sylancr eqeltrid ancoms sylan2
+      unexg 3adant2 sbthlem9 3adant1 f1oen3g syl2anc exlimivv sylbi ) CLMZBCNOZ
+      CBNOZPZVSBCEQZUAZCBFQZUAZPZFRERZBCUBOZVSWDWFSZSZFRERVSWJFRERZSZWHWBVSWJEF
+      UCWGWKEFVSWDWFUDUEWBVSVTWASZSWMVSVTWAUDWNWLVSWNWDERZWFFRZSWLVTWOWAWPBCEKU
+      FCBFHUFUGWDWFEFUHUIUJUSUKWGWIEFWGGTMZBCGULZWIVSWFWQWDWFVSWECUMZWQCBWEUNWS
+      VSWQWSVSSZGWCDUOZUPZWEUQZBXAURZUPZUTZTJWTXBTMXETMZXFTMWCXAEVAVBWTWELMXCLM
+      XGCWEVCWEVDXCXDLVEVFXBXETTVKVGVHVIVJVLWDWFWRVSABCDEFGHIJVMVNBCGTVOVPVQVR
+      $.
+    $( $j usage 'sbthfilem' avoids 'ax-pow'; $)
+  $}
+
+  ${
+    $d B w $.  $d A w z $.  $d f g w x y z $.
+    $( Schroeder-Bernstein Theorem for finite sets, proved without using the
+       Axiom of Power Sets (unlike ~ sbth ).  (Contributed by BTernaryTau,
+       4-Nov-2024.) $)
+    sbthfi $p |- ( ( B e. Fin /\ A ~<_ B /\ B ~<_ A ) -> A ~~ B ) $=
```

Comment:
iset.mm has had a very similar theorem since 2021: https://us.metamath.org/ileuni/fisbth.html

I suspect the `𝐴 ∈ Fin` condition is likely needed in iset.mm (because it doesn't have ssfi).

On the flip side, iset.mm doesn't (at least so far) make any attempt to avoid the Axiom of Power Sets, and I haven't done any investigation of whether that would be feasible.